libs/common/src/common/config/app.py

```diff
@@ -125,10 +125,12 @@
 
 lock_store_type = _config("LOCK_STORE_TYPE", default="in_memory")
 
+use_redis = _config("USE_REDIS", default=False, cast=bool)
 redis_hostname = _config("REDIS_HOSTNAME", default=None)
-redis_port = _config("REDIS_PORT", default=None)
+redis_port = _config("REDIS_PORT", default=6379, cast=int)
 redis_username = _config("REDIS_USERNAME", default=None)
 redis_password = _config("REDIS_PASSWORD", default=None)
+redis_db = _config("REDIS_DB", default=0, cast=int)
```

Comment:
I suppose this should be a string?

Reply:
I think it can be both, but I changed it to a `str` eitherways. :+1: 
[Here](https://github.com/redis/redis-py/blob/master/redis/asyncio/client.py#L209)

Reply:
Sounds fair, I though this was actually the host